### PR TITLE
Added Uncles RPC methods

### PR DESCRIPTION
**Pull request by shunjizhan**
## Change
block scout support

## Test
block scout can show most of the data

Closes: #284
Closes: #310

### CODE REVIEW

File: eth-providers/src/base-provider.ts [hunk 1]
```diff
@@ -263,13 +256,11 @@ export abstract class BaseProvider extends AbstractProvider {
 
         this._listeners[NEW_LOGS]?.forEach(({ cb, filter }) => {
           const filteredLogs = logs.filter((l) => filterLog(l, filter));
-          filteredLogs.forEach((l) =>
+          const response = hexlifyRpcResult(filteredLogs);
+          response.forEach((l: any) =>
```
Comment (shunjizhan) [inline eth-providers/src/base-provider.ts:260]: `filteredLogs.forEach` ?

**Comment by shunjizhan** [comment]
looks good to me

**Review by ntduan — APPROVED** [review]

File: eth-providers/src/base-provider.ts [hunk 2]
```diff
@@ -424,32 +412,81 @@ export abstract class BaseProvider extends AbstractProvider {
...
...04051905081810181811067ffffffffffffffff8211171561167b5761167a6117f4565b5b8060405250919050565b600067ffffffffffffffff8211156116a05761169f6117f4565b5b601f19601f8301169050602081019050919050565b600081519050919050565b600081519050919050565b600081905092915050565b600082825260208201905092915050565b60006116f28261177b565b91506116fd8361177b565b9250827fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff03821115611732576117316117c5565b5b828201905092915050565b60006117488261175b565b9050919050565b60008115159050919050565b600073ffffffffffffffffffffffffffffffffffffffff82169050919050565b6000819050919050565b600060ff82169050919050565b60005b838110156117b0578082015181840152602081019050611795565b838111156117bf576000848401525b50505050565b7f4e487b7100000000000000000000000000000000000000000000000000000000600052601160045260246000fd5b7f4e487b7100000000000000000000000000000000000000000000000000000000600052604160045260246000fd5b6000601f19601f8301169050919050565b61183d8161173d565b811461184857600080fd5b50565b6118548161177b565b811461185f57600080fd5b50565b61186b81611785565b811461187657600080fd5b5056fea2646970667358221220416663509542905b08b5355f863b01962ffe404f8af4539192b7aecad7247fa864736f6c63430008000033"
```
Comment (shunjizhan) [inline eth-providers/src/base-provider.ts:465]: maybe we can put this into `consts`?

**Review by zjb0807 — APPROVED** [review]
tested on karura testnet.

**Pull request merged**